server/asset/dcr/dcr_test.go

```diff
@@ -1090,18 +1090,13 @@ func TestUTXOs(t *testing.T) {
 		t.Fatalf("case 13 - received error for utxo: %v", err)
 	}
 
-	contract := &Contract{Output: utxo.Output}
-
 	// Now try again with the correct vout.
-	err = contract.auditContract() // sets refund and swap addresses
+	contract, err := auditContract(utxo.Output) // sets refund and swap addresses
 	if err != nil {
 		t.Fatalf("case 13 - unexpected error auditing contract: %v", err)
 	}
-	if contract.SwapAddress() != swap.recipient.String() {
-		t.Fatalf("case 13 - wrong recipient. wanted '%s' got '%s'", contract.SwapAddress(), swap.recipient.String())
-	}
-	if contract.RefundAddress() != swap.refund.String() {
-		t.Fatalf("case 13 - wrong recipient. wanted '%s' got '%s'", contract.RefundAddress(), swap.refund.String())
```

Comment:
The `RefundAddress` method remained unused, so I didn't replicate it as a field. Can always add it later.